shared-module/synthio/MidiTrack.c

```diff
@@ -28,118 +28,109 @@
 #include "shared-bindings/synthio/MidiTrack.h"
 
 
-STATIC NORETURN void raise_midi_stream_error(uint32_t pos) {
-    mp_raise_ValueError_varg(translate("Error in MIDI stream at position %d"), pos);
+STATIC void print_midi_stream_error(synthio_miditrack_obj_t *self) {
+    mp_cprintf(&mp_plat_print, translate("Error in MIDI stream at position %d"), self->pos);
```

Comment:
I'm not sure you want to print. Instead, maybe just an `error_offset` attribute on the object that is set to the position when an error occurs. `None` if all is ok.

Reply:
An exception cannot be raised at this point, because the context is the background task that updates. Are you suggesting to add an `error_offset` property to the MidiTrack object, to let the information be retrieved programmatically? I can do that. Is it in lieu of printing this message?

Reply:
OK, I added an error_offset property and removed the message.